themes/digital.gov/src/scss/new/_uswds-overrides.scss

```diff
@@ -50,6 +50,6 @@
 }
 
 // set margin-top to 0 to leave no space between banner and alert
-.usa-site-alert.usa-site-alert--info .usa-alert {
+.usa-site-alert .usa-alert {
```

Comment:
Now we can target our class directly with `.dg-site-alert .usa-alert` instead of adding a custom override. This might require a `dg-site-alert` partial, instead of using overrides.

Alternatively, we could try a utility class like `margin-top-0` on the component.

Reply:
I tried the utility class, not specific enough so I created a `_sitewide-alert.scss` partial. Might need it in the future too.